app/Repositories/Activity/ActivityRepository.php

```diff
@@ -6,6 +6,7 @@
 use App\Models\Activity;
 use App\Models\Playlist;
 use App\Models\Project;
+use App\Models\Organization;
```

Comment:
It seems "use App\Models\Organization;" was added in ActivityRepository.php
but was not used.